capstone360/__init__.py

```diff
@@ -0,0 +1,20 @@
+# main file for server management
```

Comment:
Do we need this file here right now? Didn't we remove this? I think we may need to move everything back into capstone360, but for now, no.